src/main/java/teammates/logic/automated/FeedbackSessionClosedMailAction.java

```diff
@@ -0,0 +1,85 @@
+package teammates.logic.automated;
+
+import java.util.HashMap;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.util.Assumption;
+import teammates.common.util.Const;
+import teammates.common.util.Const.ParamsNames;
+import teammates.common.util.EmailWrapper;
+import teammates.common.util.HttpRequestHelper;
+import teammates.logic.core.EmailGenerator;
+import teammates.logic.core.FeedbackSessionsLogic;
+
+public class FeedbackSessionClosedMailAction extends EmailAction {
+
+    private String feedbackSessionName;
+    private String courseId;
+    
+    public FeedbackSessionClosedMailAction(HttpServletRequest request) {
+        super(request);
+        initializeNameAndDescription();
+        
+        feedbackSessionName = HttpRequestHelper
+                .getValueFromRequestParameterMap(request, ParamsNames.EMAIL_FEEDBACK);
+        Assumption.assertNotNull(feedbackSessionName);
+        
+        courseId = HttpRequestHelper
+                .getValueFromRequestParameterMap(request, ParamsNames.EMAIL_COURSE);
+        Assumption.assertNotNull(courseId);
+    }
+    
+    public FeedbackSessionClosedMailAction(HashMap<String, String> paramMap) {
+        super();
+        initializeNameAndDescription();
+        
+        feedbackSessionName = paramMap.get(ParamsNames.EMAIL_FEEDBACK);
+        Assumption.assertNotNull(feedbackSessionName);
+        
+        courseId = paramMap.get(ParamsNames.EMAIL_COURSE);
+        Assumption.assertNotNull(courseId);
+    }
+    
+    @Override
+    protected void doPostProcessingForSuccesfulSend() {
+        /* 
+         * Empty because no action is required on successful
+         * sending of feedback session closing mails
+         */
+    }
+
+    @Override
+    protected List<EmailWrapper> prepareMailToBeSent() {
+        
+        FeedbackSessionAttributes feedbackObject = FeedbackSessionsLogic.inst()
+                .getFeedbackSession(feedbackSessionName, courseId);
+        log.info("Fetching feedback session object for feedback session name : "
+                 + feedbackSessionName + " and course : " + courseId);
```

Comment:
Even if this was on the code on similar actions, I don't see the point of logging here. Can remove this?